hcloud/schema/pricing.go

```diff
@@ -97,7 +97,7 @@ type PricingGetResponse struct {
 // PricingPrimaryIPTypePrice defines the schema of pricing information for a primary IP
 // type at a datacenter.
 type PricingPrimaryIPTypePrice struct {
-	Datacenter   string `json:"datacenter"`
+	Datacenter   string `json:"location"`
```

Comment:
I think it is a bit suprising the get a location, while the attribute is named `Datacenter`. I get that renaming the attribute would break downward compatibilities, but existing code might also break if it now gets a location instead of a datacenter.

If downward compatibility should be preserved, my suggestion would be to add a second attribute called `Location`. Then add a comment to `Datacenter`, marking it as deprecated, if its not provided by the API.

Reply:
I agree with @jangraefen.

To make this compatible with tooling, the comment should start with `Deprecated: ...`.

Please also make sure to add the new field to `hcloud.PrimaryIPPricing` and `hcloud.PricingFromSchema()`.

Reply:
I have added the separate key and deprecated `Datacenter`.